src/legacy/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -54,271 +40,59 @@ class DashboardListingUi extends React.Component {
-    return (
-      <EuiOverlayMask>
-        <EuiConfirmModal
-          title={
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteSelectedDashboardsConfirmModal.title"
-              defaultMessage="Delete selected dashboards?"
-            />
-          }
-          onCancel={this.closeDeleteModal}
-          onConfirm={this.deleteSelectedItems}
-          cancelButtonText={
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteSelectedDashboardsConfirmModal.cancelButtonLabel"
-              defaultMessage="Cancel"
-            />
-          }
-          confirmButtonText={
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteSelectedDashboardsConfirmModal.confirmButtonLabel"
-              defaultMessage="Delete"
-            />
-          }
-          defaultFocusedButton="cancel"
-        >
-          <p>
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteDashboardsConfirmModalDescription"
-              defaultMessage="You can't recover deleted dashboards."
-            />
-          </p>
-        </EuiConfirmModal>
-      </EuiOverlayMask>
-    );
-  }
-
-  renderListingLimitWarning() {
-    if (this.state.showLimitError) {
-      return (
-        <React.Fragment>
-          <EuiCallOut
-            title={
-              <FormattedMessage
-                id="kbn.dashboard.listing.listingLimitExceededTitle"
-                defaultMessage="Listing limit exceeded"
-              />
-            }
-            color="warning"
-            iconType="help"
-          >
-            <p>
-              <FormattedMessage
-                id="kbn.dashboard.listing.listingLimitExceededDescription"
-                defaultMessage="You have {totalDashboards} dashboards, but your {listingLimitText} setting prevents
-                the table below from displaying more than {listingLimitValue}. You can change this setting under {advancedSettingsLink}."
-                values={{
-                  totalDashboards: this.state.totalDashboards,
-                  listingLimitValue: this.props.listingLimit,
-                  listingLimitText: (
-                    <strong>
-                      listingLimit
-                    </strong>
-                  ),
-                  advancedSettingsLink: (
-                    <EuiLink href="#/management/kibana/settings">
-                      <FormattedMessage
-                        id="kbn.dashboard.listing.listingLimitExceeded.advancedSettingsLinkText"
-                        defaultMessage="Advanced Settings"
-                      />
-                    </EuiLink>
-                  )
-                }}
-              />
-            </p>
-          </EuiCallOut>
-          <EuiSpacer size="m" />
-        </React.Fragment>
-      );
-    }
   }
 
-  renderNoResultsMessage() {
-    if (this.state.isFetchingItems) {
-      return '';
-    }
-
+  render() {
+    const { intl } = this.props;
```

Comment:
AFAIK the I18N hoc shouldn't be used anymore and should be replaced by a direkt `i18n.translate` call. @timroes could you elaborate?

Reply:
I also remember seeing this, but I'm not sure where. Couldn't find this request in the .MDs

Anyway, I changed it.